onecodex/viz/__init__.py

```diff
@@ -96,8 +96,24 @@ def onecodex_theme():
     }
 
 
-def configure_onecodex_theme(altair_module=None):
-    """Configure One Codex Altair theme."""
+def onecodex_renderer(spec: dict, **metadata) -> dict:
+    """Render spec as HTML and optionally SVG if vl-convert is installed."""
+    from altair.vegalite.v5.display import html_renderer, svg_renderer
+
+    bundle = html_renderer(spec, **metadata)
+
+    try:
+        import vl_convert  # noqa
+    except ImportError:
+        pass
+    else:
+        bundle.update(svg_renderer(spec, **metadata))
+
+    return bundle
+
+
+def configure_onecodex_theme_and_renderer(altair_module=None):
```

Comment:
I'll change the name back. I think it's ok to configure the renderer in this function because we were doing that previously (setting `altair_saver` or `html` renderer in this function)